src/Tools/dotnet-monitor/Exceptions/ExceptionsOperation.cs

```diff
@@ -126,6 +138,25 @@ private static async Task WriteTextInstance(Stream stream, IExceptionInstance in
                     instance.TypeName,
                     instance.Message));
 
+            if (instance.CallStackResult.Stacks.Any())
+            {
+                CallStack stack = instance.CallStackResult.Stacks.First(); // We know the result only has a single stack
+
+                foreach (CallStackFrame frame in stack.Frames)
+                {
+                    Monitoring.WebApi.Models.CallStackFrame frameModel = StacksFormatter.CreateFrameModel(frame, instance.CallStackResult.NameCache);
+
+                    await writer.WriteLineAsync(
+                        string.Format(
+                            CultureInfo.InvariantCulture,
+                            Strings.OutputFormatString_FirstChanceExceptionStackFrame,
+                            frameModel.ClassName,
+                            frameModel.MethodName));
+                }
+
+                writer.WriteLine();
```

Comment:
If this is to provide an empty newline between exception instances, then this should be outside of the if condition.